cliTest/testaxialparser.cpp

```diff
@@ -31,7 +31,9 @@ TEST_CASE("Test mode and help")
                                 " Further flags for axial analysis are:\n"\
                                 "   -xac Include choice (betweenness)\n"\
                                 "   -xal Include local measures\n"\
-                                "   -xar Inlcude RA, RRA and total depth\n\n");
+                                "   -xar Inlcude RA, RRA and total depth\n"\
```

Comment:
Fix the typo in `inLCude` when touching this area anyway? It's my favourite typo, I do it all the time...